entrypoint.sh

```diff
@@ -1,9 +1,10 @@
-#!/bin/bash
+#!/bin/sh
 set -e
 
 echo "Starting the Jekyll Action"
 
 echo "::debug ::Starting bundle install"
+bundle config path vendor/bundle
```

Comment:
👍

Reply:
This is can be collapsed into one with the single line below, like this:

```
bundle install --path vendor/bundle
```

Reply:
The `install --path` option is being deprecated in favor on the `config path` command.

Reply:
Okay thanks. I'll start following in the new pattern in my projects too then.